chart/openfaas/README.md

```diff
@@ -249,8 +249,9 @@ Additional OpenFaaS options in `values.yaml`.
 | `serviceType` | Type of external service to use `NodePort/LoadBalancer` | `NodePort` |
 | `basic_auth` | Enable basic authentication on the Gateway | `true` |
 | `rbac` | Enable RBAC | `true` |
-| `httpProbe` | Setting to true will use HTTP for readiness and liveness probe on the OpenFaaS system Pods (incompatible with Istio < 1.1.5) | `true` |
+| `psp` | Enable Pod Security Policy for OpenFaaS accounts | `false` |
 | `securityContext` | Deploy with a `securityContext` set, this can be disabled for use with Istio sidecar injection | `true` |
+| `httpProbe` | Setting to true will use HTTP for readiness and liveness probe on the OpenFaaS system Pods (incompatible with Istio < 1.1.5) | `true` |
```

Comment:
Please rebase this with my latest change which says the same in the positive vs negative.

compatible with Istio >= 1.1.5

Reply:
Done